releasenotes/notes/1.3/backend-sampler-v2-level1-dc13af460cd38454.yaml

```diff
@@ -1,15 +1,15 @@
 ---
 features:
   - |
-    Support for level 1 data was added to :class:`~.BackendSamplerV2` as was
+    Support for level 1 data was added to :class:`~.BackendSamplerV2`, as was
     support for passing options through to the ``run()`` method of the wrapped
-    :class:`~.BackendV2`. The run options can be specified using a
-    ``"run_options"`` entry inside of the ``options`` dicitonary passed to
+    :class:`~.BackendV2`. The run options can be specified by using a
+    ``"run_options"`` entry inside the ``options`` dictionary passed to
     :class:`~.BackendSamplerV2`. The ``"run_options"`` entry should be a
-    dictionary mapping argument names to values for passing to the backend's
-    ``run()`` method. When a ``"meas_level"`` option with a value of 1 is set
+    dictionary that maps argument names to values to be passed to the backend's
+    ``run()`` method. When a ``"meas_level = 1 "`` is set
     in the run options, the results from the backend will be treated as level 1
-    results rather as bit arrays (the level 2 format).
+    results instead of bit arrays (the level 2 format).
 upgrade:
```

Comment:
Is this really an upgrade note of a fix note? If it's an upgrade note we should explain the rationale behind the change, and also change the category to `upgrade_primitives`.